model/src/main/kotlin/org/digma/intellij/plugin/model/rest/recentactivity/RecentActivityResult.kt

```diff
@@ -19,9 +19,10 @@ constructor(
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class RecentActivityResponseEntry
 @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
-@ConstructorProperties("environment", "traceFlowDisplayName", "firstEntrySpan", "lastEntrySpan", "latestTraceId", "latestTraceTimestamp", "latestTraceDuration", "slimAggregatedInsights")
+@ConstructorProperties("environment", "environmentId", "traceFlowDisplayName", "firstEntrySpan", "lastEntrySpan", "latestTraceId", "latestTraceTimestamp", "latestTraceDuration", "slimAggregatedInsights")
 constructor(
         val environment: String,
+        val environmentId: String,
```

Comment:
what is environment and why do we need both environment and environmenId ?
I think we should use only environemntId all over.
with environmentId you can use
org.digma.intellij.plugin.analytics.EnvUtilsKt#getEnvironmentNameById
org.digma.intellij.plugin.analytics.EnvUtilsKt#getEnvironmentById